### PR TITLE
[TASK] Document the licensing in the README

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+
+This TYPO3 extension is an example for writing TYPO3 extensions.
+Is is used in the trainings provided by the TYPO3 Trainer Network.
+
+## License
+
+The code is licensed under the [Gnu General Public License V2](LICENSE).
+
+The extension icon is taken from [Font Awesome](https://fontawesome.com/)
+and is licensed under the [CC BY 4.0 License](https://fontawesome.com/license/free).
```
Comment (andreaswolf) [inline README.md:18]: Isn't it "CC-BY"?

Comment (oliverklee) [reply README.md:18]: I'll look it up …

Comment (oliverklee) [reply README.md:18]: According to https://creativecommons.org/licenses/by/4.0/deed.en, there is no hyphen.

**Comment by oliverklee** [comment]
Rebased and repushed.

File: README.md (hunk 1 above)
Comment (andreaswolf) [reply README.md:18]: ah, there is a hyphen only if there are multiple limitations, e.g. in "CC BY-SA".

**Review by andreaswolf — APPROVED** [review]

**Pull request merged**